examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -55,10 +50,5 @@ public void testE2EWordCount() throws Exception {
         options.getJobName(), "output", "results"}));
```

Comment:
If we rely on the jobname, we will still be specific to Dataflow.
I believe you'll need to use a UUID or something equivalent